src/dist/replication/lib/replica_2pc.cpp

```diff
@@ -104,6 +123,7 @@ void replica::init_prepare(mutation_ptr &mu, bool reconciliation)
 
     error_code err = ERR_OK;
     uint8_t count = 0;
+    const auto request_count = mu->client_requests.size();
```

Comment:
I think you can move this line to L153

Reply:
There are a goto sentence on L151, varieties should be defined before goto.